### ISSUE TITLE
False positive: `scss/declaration-property-value-no-unknown` throws parse error on Polymer mixins and namespaced functions

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint-scss/stylelint-scss#1288]
### What minimal example or steps are needed to reproduce the bug?

**Namespaced module function inside a CSS property:**
```scss
.element {
  background-image: linear-gradient(
    45deg,
    css-vars.plx-var(background, hover) 20%,
    transparent 20%
  );
}
```

### Which Stylelint-related dependencies are you using?
stylelint: 17.12.0

stylelint-scss: 6.11.1

### What did you expect to happen?
The linter should recognize this as non-standard or valid custom syntaxes (SCSS module namespaces) and either ignore it or pass validation.

### What actually happened?
The `css-tree` parser fails to parse this function, resulting in a rejectedParseError and causing the linter to fail with the following false positives:
```
Cannot parse property value "linear-gradient( 45deg, css-vars.plx-var(background, hover) 20%, transparent 20% )" for property "background-image" (scss/declaration-property-value-no-unknown)
```

### Do you have a proposal to fix the bug?
Yes. We can add  anearly return in the rule before cssTree.parse is called to intercept these specific non-standard formats before the parser chokes on it.

Adding the following helper functions and checks resolves the issue:

```JavaScript
// Add this check before `cssTree.parse(value, ...)`
if (containsNamespacedFunction(value)) return;

// Helper function to catch namespaced module functions `namespace.function()`
/**
 * Check if a value contains a namespaced function call.
 * @param {string} value
 * @returns {boolean}
 */
function containsNamespacedFunction(value) {
  return /[\w-]+\.[\w-]+\(/.test(value);
}
```
